ecommerce_integrations/unicommerce/delivery_note.py

```diff
@@ -0,0 +1,115 @@
+import frappe
+from ecommerce_integrations.unicommerce.constants import (
+    CHANNEL_ID_FIELD,
+    CHANNEL_TAX_ACCOUNT_FIELD_MAP,
+    FACILITY_CODE_FIELD,
+    IS_COD_CHECKBOX,
+    MODULE_NAME,
+    ORDER_CODE_FIELD,
+    ORDER_ITEM_BATCH_NO,
+    ORDER_ITEM_CODE_FIELD,
+    ORDER_STATUS_FIELD,
+    PACKAGE_TYPE_FIELD,
+    SETTINGS_DOCTYPE,
+    TAX_FIELDS_MAPPING,
+    TAX_RATE_FIELDS_MAPPING,
+)
+from ecommerce_integrations.unicommerce.api_client import UnicommerceAPIClient
+from ecommerce_integrations.unicommerce.utils import create_unicommerce_log
+SHIPMENT_STATES = [
+    "CREATED",
+    "LOCATION_NOT_SERVICEABLE",
+    "PICKING",
+    "PICKED",
+    "PACKED",
+    "READY_TO_SHIP",
+    "CANCELLED",
+    "MANIFESTED",
+    "DISPATCHED",
+    "SHIPPED",
+    "DELIVERED",
+    "PENDING_CUSTOMIZATION",
+    "CUSTOMIZATION_COMPLETE",
+    "RETURN_EXPECTED",
+    "RETURNED",
+    "SPLITTED",
+    "RETURN_ACKNOWLEDGED",
+    "MERGED",
+]
+
+import time
+
+@frappe.whitelist()
+def prepare_delivery_note():
+    try:
+        time.sleep(15)
```

Comment:
what is this?

Reply:
can you summarize in PR why delivery notes are generated and why it can't be done with default behaviour?

Reply:
Removed